### ISSUE TITLE
Flyout rendering 

### ISSUE THREAD

**Issue by eduardoagr** [dotnet/maui#25053]
### Description

Maui on the Windows platform, when started in a way that Flyout is invisible, if the width of Flyout is less than the width required by Flyout Item, the renderer only renders the content with the set width., and if you reset the with programmatically, with a button that you toggle, saving the state of the with, the app don't renter correctly

For example, 

if You create an app, and establish the flyout behavior on mobile Disabled, and for desktop Locked, 

And save the current with in storage, when you reset the app, and et the flyout to normal, the items don't render

### Steps to Reproduce

1. Create a Maui app
2. Create two pages with titles (Important)
3. Create a Shell view Model
4. Create a flyout header
5. Out a button to toggle to turn of compact view (65) and normal (320)
6. Make the flyout disabled for mobile and locked in desktop
7. In the view Model, create a method to save in preferences, the width of the flyout
8. Start the app
9. Activate compact mode, making the width of the flyout 65 
10. Restart the app
11. Turn of compact mode
12. Titles don't appear (Bug)

### Version with bug

8.0.90 SR9

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

windows10.0.19041.0

### Did you find any workaround?

No

**Comment by drasticactions** [comment]
If you could please make a reproduction project of your steps, debugging and stepping through your issue would be much easier. I know it's inconvenient, but it would be a big help.

**Comment by eduardoagr** [comment]
I now sorry, well this is happening in my app, so I don't know if this is an inconvenience 

https://github.com/eduardoagr/METROWIND

can you please tell me when you cloned the repo, so I can put it private 

look at this weird problem

https://reccloud.com/u/p2k3tzl

Microsoft QA told me, that it was a rendering problem

I hope you do not mind that I left license keys in there

**Comment by eduardoagr** [comment]
Since this is Microsoft repo, my keys are safe right

**Comment by OvrBtn** [comment]
@eduardoagr if you are talking about the keys that are hard coded in one of the classes, no they are not. 
Open browser in private mode and open the repo and you will be able to see everything, that means that everyone can too.

If those are your real keys, make the repo private asap.

**Comment by eduardoagr** [comment]
Yes, I know I just put it public so you can see the error. Did you see the error, can I put it private again

**Comment by eduardoagr** [comment]
Hello, did you the error, what is the status, this is important for me

**Comment by eduardoagr** [comment]
Any Update

**Comment by Zhanglirong-Winnie** [comment]
Hi, @eduardoagr 
I want to verify this issue, but I don't have permission to view it.
If possible, can you release the permission or provide it again? 
Looking forward to your reply.

**Comment by eduardoagr** [comment]
I jus put it on public

**Comment by eduardoagr** [comment]
be sure to clone the branch metrowind 1.1 and, letme know, to put it private again

**Comment by eduardoagr** [comment]
I just update it

**Comment by eduardoagr** [comment]
clone it and let me now when you cloned it

**Comment by Zhanglirong-Winnie** [comment]
I cloned it, you can close it. And I can't see the screen recording you provided. If it's convenient, can you share another copy with me? Thank you

**Comment by eduardoagr** [comment]
Sorry about that, and thank you for hearing my cry, I tried to fix it myself by forcing the flyout to render, but if I do that, I get some weird flickering problems

https://reccloud.com/u/34lh9zb

please let me know what is the error

**Comment by Zhanglirong-Winnie** [comment]
This issue has been verified using  Visual Studio 17.12.0 Preview 2.1(8.0.82 & 8.0.91 & 8.0.90). Can repro this issue on windows platform.

**Comment by eduardoagr** [comment]
what does this mean
